src-opam/distro.ml

```diff
@@ -187,6 +187,8 @@ let win10_kb_number_to_lcu (v : win10_release) kb =
   | lcu -> Some (v, lcu)
   | exception Not_found -> None
 
+type macos_all = [ `Latest | `Macos_monterey ]
```

Comment:
I think using their version number is much easier to maintain and remember which version they are
```suggestion
type macos_all = [ `Latest | `V12 ]
```